homeassistant/components/escea/config_flow.py

```diff
@@ -10,28 +11,39 @@
 from homeassistant.helpers import config_entry_flow
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 
-from .const import DISPATCH_CONTROLLER_DISCOVERED, ESCEA, TIMEOUT_DISCOVERY
+from .const import (
+    DISPATCH_CONTROLLER_DISCOVERED,
+    ESCEA,
+    ESCEA_FIREPLACE,
+    TIMEOUT_DISCOVERY,
+)
 from .discovery import async_start_discovery_service, async_stop_discovery_service
 
 _LOGGER = logging.getLogger(__name__)
 
 
-async def _async_has_devices(hass: HomeAssistant):
+async def _async_has_devices(hass: HomeAssistant) -> bool:
 
     controller_ready = asyncio.Event()
+    remove_handler: Callable[[], None]
```

Comment:
We don't need this type. `async_dispatcher_connect` is typed so we know the return value of that call.

Reply:
Fixed, thanks again